argo-cd-apps/base/all-clusters/infra-deployments/tracing-workload-otel-collector/tracing-workload-otel-collector.yaml

```
@@ -3,121 +3,45 @@ kind: ApplicationSet
-                batch: {}
-                # If set to null, will be overridden with values based on k8s resource limits
-                memory_limiter:
-                  check_interval: 2s
-                  limit_mib: 512
-                  spike_limit_percentage: 100
-                attributes/collector-info:
-                  actions:
-                    - key: collector.hostname
-                      action: insert
-                      value: ${env:HOSTNAME}
-                    - key: collector.clustername
-                      action: insert
-                      value: dev
-              receivers:
-                jaeger: null
-                prometheus: null
-                zipkin: null
-                otlp:
-                  protocols:
-                    grpc:
-                      endpoint: 0.0.0.0:4317
-                      max_recv_msg_size_mib: 999999999
-                    http:
-                      endpoint: 0.0.0.0:4318
-              service:
-                extensions:
-                  - health_check
-                  - memory_ballast
-                pipelines:
-                  traces:
-                    exporters:
-                      - debug
-                    processors:
-                      - memory_limiter
-                      - attributes/collector-info
-                      - batch
-                    receivers:
-                      - otlp
-                  metrics: null
-            # Configuration for ports
-            ports:
-              otlp:
-                enabled: true
-                containerPort: 4317
-                servicePort: 4317
-                hostPort: 4317
-                protocol: TCP
-              otlp-http:
-                enabled: true
-                containerPort: 4318
-                servicePort: 4318
-                hostPort: 4318
-                protocol: TCP
-              jaeger-compact:
-                enabled: false
-              jaeger-thrift:
-                enabled: false
-              jaeger-grpc:
-                enabled: false
-              zipkin:
-                enabled: false
-
+      sources:
+          # Chart from OTel Project
+        - chart: opentelemetry-collector
+          repoURL: https://open-telemetry.github.io/opentelemetry-helm-charts
+          targetRevision: 0.90.1
+          helm:
+            parameters:
+              - name: "image.repository"
+                value: otel/opentelemetry-collector-k8s
+              - name: "mode"
+                value: deployment
+            valueFiles:
+            - $values/{{values.sourceRoot}}/{{values.environment}}/otel-collector-helm-values.yaml
+          # Values for chart from git
+        - repoURL: https://github.com/redhat-appstudio/infra-deployments.git
+          targetRevision: main
+          ref: values
       destination:
         namespace: konflux-otel
-        server: '{{server}}'
+        server: https://kubernetes.default.svc
```

we use application generators(line 12 to 19) so this must be kept to '{{server}}' variable to create the application on the right target cluster.

```suggestion
        server: '{{server}}'
```

---

Fixed.